tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -133,7 +133,13 @@ public function dataFileAsserts(): iterable
 
 		yield from $this->gatherAssertTypes(__DIR__ . '/../Reflection/data/staticReturnType.php');
 
-		yield from $this->gatherAssertTypes(__DIR__ . '/data/minmax-arrays.php');
```

Comment:
turned the existing assertions ingo a php<8 case (`minmax-arrays.php`)

copied the existing assertion files and adjusted them for php8+ (`minmax-php8.php`)

extracted the assertions which are php version independent into `minmax.php`